GetIntoTeachingApi/AppStart/Prometheus/PrometheusMetricLabels.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using GetIntoTeachingApi.Utils;
+using Prometheus;
+
+namespace GetIntoTeachingApi.AppStart.Prometheus
+{
+    public static class PrometheusMetricLabels
```

Comment:
May be worth merging the Utils/ApplicationExtensions which does prometheus config into here as well